composer.json

```diff
@@ -24,8 +24,10 @@
         }
     },
     "require": {
+        "vimeo/payment-gateway-logger": "^1.0",
         "omnipay/common": "~2.0",
-        "composer/ca-bundle": "^1.1"
+        "composer/ca-bundle": "^1.1",
+        "ext-soap": "*"
```

Comment:
what's this for?

Reply:
It's a safety thing to ensure the PHP soap extension is installed. From here https://getcomposer.org/doc/04-schema.md#package-links

> It is important to list PHP extensions your project requires. Not all PHP installations are created equal: some may miss extensions you may consider as standard (such as ext-mysqli which is not installed by default in Fedora/CentOS minimal installation systems). Failure to list required PHP extensions may lead to a bad user experience: Composer will install your package without any errors but it will then fail at run-time. The composer show --platform command lists all PHP extensions available on your system. You may use it to help you compile the list of extensions you use and require. Alternatively you may use third party tools to analyze your project for the list of extensions used.

`*` is the preferred versioning way of installing due to inconsistent versioning.

Reply:
oh ok, so not actually related to the logging, right?

Reply:
Correct. There are similar issues with curl and json. We need to explicitly tell composer that `ext-curl` and `ext-curl` are being used in this package.